src/Compilers/VisualBasic/Test/Semantic/Semantics/AccessCheck.vb

```diff
@@ -1864,10 +1864,10 @@ End Class
 
             CompilationUtils.AssertTheseDiagnostics(compilation,
 <expected>
-BC30508: 'B' cannot expose type 'A.B(Of T).C' in class 'A' through class 'B'.
+BC30508: 'B' cannot expose type 'A.B(Of T As A.B(Of T).C).C' in class 'A' through class 'B'.
```

Comment:
Is this change expected? The resulting error message is harder to read.

Reply:
Please see my comment above explaining why this happens and a possible alternative. I can go either way but wasn't sure about the broader impact of changing a predefined format.